clang/test/CIR/CodeGen/dlti.c

```diff
@@ -0,0 +1,23 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -emit-cir %s -o %t.cir
```

Comment:
Can you add another pair of RUN lines to test with a big-endian target?

Reply:
> If I add a big-endian target triple, it might cause errors for others who haven't built LLVM with support for that architecture. Is there a way when the target isn't available?

You can add it in a separate test and use something like `// REQUIRES: aarch64-registered-target` (or whichever target you're testing).